### DISCUSSION TITLE
Bring back secondary Y axis option as it was in old graph panel (feature request)

### DISCUSSION THREAD

**Discussion by barts2108** [grafana/grafana#66908] [Dashboards, Panels & Visualizations]
In the old Graph panel, it was possible to click the colored line from a legen and move that metric to the secondary Y-axis.

![image](https://user-images.githubusercontent.com/520875/233276073-b0615f9c-8e4b-4868-a686-65ef0e6f86cf.png)

![image](https://user-images.githubusercontent.com/520875/233276112-f16b49e6-ff37-42ec-a325-2fea3cfd6a13.png)

In the new Time Series panel this option is not available

![image](https://user-images.githubusercontent.com/520875/233276312-105418bf-5a79-476b-91c9-c18eff6a71d2.png)

It is only possible to add an override, and to do so you need to have editor rights. 
Furthermore, the old option is temporary, and easy to use in case you want to move one (or more) signals to the secondary y-axis, which can be used by an "operator". The new option is more complex, is permantent, and needs an "engineer" to make the changes.

Can the old option be implemented again as it used to be (alongside with the override). Then the "engineer" can create the (provisioned) dashboards as they were meant to be, and the "operator" can move a metric temporary.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82733. Feel free to continue the discussion around this there. Thank you!
